apps/web/src/modules/job/components/card/type.ts

```diff
@@ -0,0 +1,35 @@
+export interface cardProps {
```

Comment:
We should separate type file for each component
(Header/types.ts , Footer/types.ts)